Code/EnginePlugins/OpenXRPlugin/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+ez_cmake_init()
+
+ez_requires_openxr()
+
+# Get the name of this folder as the project name
+get_filename_component(PROJECT_NAME ${CMAKE_CURRENT_SOURCE_DIR} NAME_WE)
+
+ez_create_target(LIBRARY ${PROJECT_NAME})
+
+target_link_libraries(${PROJECT_NAME}
+  PUBLIC
+  GameEngine
+)
+
+target_compile_definitions(${PROJECT_NAME} PUBLIC BUILDSYSTEM_ENABLE_OPENXR_SUPPORT)
```

Comment:
I think you can now remove BUILDSYSTEM_ENABLE_MIXEDREALITY_SUPPORT and all related code, no ?

Reply:
Ie the whole WindowsMixedReality library.

Reply:
Yes I can but will do that in a separate commit to not bloat this one even further.